tgui/packages/tgui/interfaces/common/CrewManifest.js

```diff
@@ -0,0 +1,177 @@
+import { useBackend } from "../../backend";
+import { Box, Section, Table } from "../../components";
+import { decodeHtmlEntities } from "../../../common/string";
+import { COLORS } from "../../constants";
+
+const deptCols = COLORS.department;
+
+const HeadRoles = [
+  "Captain",
+  "Head of Security",
+  "Chief Engineer",
+  "Chief Medical Officer",
+  "Research Director",
+  "Head of Personnel",
+];
+// Head colour check. Abbreviated to save on 80 char
+const HCC = role => {
+  // Return green if they are the head
+  if (HeadRoles.indexOf(role) !== -1 || role === "Internal Affairs Agent") {
+    return "green";
+  }
+  // Return yellow if its the qm
+  if (role === "Quartermaster") {
+    return "yellow";
+  }
+  // Return orange if its a regular person
+  return "orange";
+};
+
+// Head bold check. Abbreviated to save on 80 char
+const HBC = role => {
+  // Return true if they are a head, or a QM/IAA
+  if ((HeadRoles.indexOf(role) !== -1) || role === "Quartermaster" || role === "Internal Affairs Agent") {
+    return true;
+  } else {
+    return false;
+  }
+};
```

Comment:
```suggestion
const HBC = role => ((HeadRoles.indexOf(role) !== -1) || role === "Quartermaster" || role === "Internal Affairs Agent")
```
Вроде так тож будет работать